demo/meeting/back-end-resources/lambda-layer/nodejs/node_modules/utils.js

```diff
@@ -0,0 +1,82 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0var AWS = require('aws-sdk');
+
+var AWS = require('aws-sdk');
```

Comment:
Should we have this `util.js` under the `lambda-layer` or `nodejs` than the `node_modules`?

Reply:
That's where it has to go.  The directory structure for layers requires it.